modules/v2x/proto/v2x_obu_traffic_light.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto2";
+
+package apollo.obu.v2x;
+
+import "modules/common/proto/header.proto";
+
+message NextTrafficLightInfo {
+  enum Color {
+    UNKNOWN = 0;
+    RED = 1;
+    YELLOW = 2;
+    GREEN = 3;
+    BLACK = 4;
+    FLASH_GREEN = 5;
+  };
+  optional Color color = 1;
+  optional int32 start_time = 2;
+  optional int32 end_time = 3;
```

Comment:
What's the unit of start_time and end_time. Please note it explicitly. And, is init32 large enough for holding time values?